operations/overlay_compute_jaccard.yml

```diff
@@ -0,0 +1,44 @@
+id: overlay_compute_jaccard
+subclass_of: overlay
+name: overlay compute jaccard
+description: This operation computes the Jaccard similarity that measures how many intermediate_node_key's are shared in common between each start_node_key and object_node_key. And it will then create virtual edges and add edge attributes (with the property name jaccard_index) between each start_node_key and object_node_key. This is used for purposes such as "find me all drugs (start_node_key) that have many proteins (intermediate_node_key) in common with this disease (end_node_key)." This can be used for downstream filtering to concentrate on relevant bioentities.
```

Comment:
Same comment as above: I suggest the following change to the description:
"This operation computes the Jaccard similarity which measures how many intermediate_node_key's are directly connected to both the start_node_key and end_node_key for all pairs of start_node_keys and end_node_keys. It will then add edges to the knowledge graph along with edge attributes (with the property name jaccard_index) between each start_node_key and object_node_key. A query graph edge will also be added using the key specified by virtual_relation_label. This is used for purposes such as "find me all drugs (start_node_key) that have many proteins (intermediate_node_key) in common with this disease (end_node_key)." This can be used for downstream filtering to concentrate on relevant bioentities."